patches/chrome-android-java-src-org-chromium-chrome-browser-ChromeApplicationImpl.java.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ChromeApplicationImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/ChromeApplicationImpl.java
+index 346488166d3082195308895b56e69270166ea321..d7d84ac7c400a458c7b9fc90c19397f35d1e5fd2 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/ChromeApplicationImpl.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/ChromeApplicationImpl.java
+@@ -78,6 +78,7 @@ public class ChromeApplicationImpl extends SplitCompatApplication.Impl {
+             PartitionResolverSupplier.setInstance(new ProfileResolver());
+ 
+             AppHooks.get().getChimeDelegate().initialize();
++            com.wireguard.android.backend.GoBackend.setAlwaysOnCallback(new com.wireguard.android.backend.GoBackend.AlwaysOnCallback() {@Override public void alwaysOnTriggered() { org.chromium.chrome.browser.vpn.utils.BraveVpnProfileUtils.getInstance().startVpn(getApplication()); }});
```

Comment:
Is it necessary to call it at the end? If we can call in the beginning, we may add our `BraveApplicationImplBase` as a base class for this one and call it there. And change super class in bytecode.

Reply:
@samartnik we need to initialise the call inside  `SplitCompatApplication.isBrowserProcess()` So in that case it wouldn't work.